apps/site/src/app/apply/sections/Form/BasicInformation.tsx

```diff
@@ -0,0 +1,165 @@
+"use client";
+
+import { useState } from "react";
+import RadioSelect from "../Components/RadioSelect";
+import PasswordEyeProps from "../Components/PasswordEye";
+import RequiredAsterisk from "../Components/RequiredAsterisk";
+import styles from "./Form.module.scss";
+
+const pronouns = [
+	{ value: "she", text: "She/her/hers" },
+	{ value: "he", text: "He/him/his" },
+	{ value: "they", text: "They/them/theirs" },
+	{ value: "ze", text: "Ze/zir/zirs" },
+	{ value: "other", text: "Other:" },
+];
+
+const ethnicity = [
+	{ value: "American", text: "American Indian or Alaskan" },
+	{ value: "Asian", text: "Asian or Pacific Islander" },
+	{ value: "Black", text: "Black or African American" },
+	{ value: "Hispanic", text: "Hispanic" },
+	{ value: "White", text: "White or Caucasian" },
+	{ value: "Two-or-more", text: "Two or more races" },
+	{ value: "Prefer not to answer", text: "Prefer not to answer" },
+	{ value: "other", text: "Other:" },
+];
+
+export default function BasicInformation() {
+	//the password state here is supposed to be used for further validation when we
+	//check confirm password, but I'm pretty sure that will have to wait until we
+	//implement the submission process, so I just leave it here for now
+
+	//still confused about this one as conflicting opinions from Sam and Taesungh
+	const [password, setPassword] = useState("");
+
+	const [passwordVisible, setPasswordVisible] = useState(false);
+	const [confirmPasswordVisible, setConfirmPasswordVisible] = useState(false);
+
+	const handleChangePassword = () => {
+		setPasswordVisible(!passwordVisible);
+	};
+
+	const handleChangeConfirmPassword = () => {
+		setConfirmPasswordVisible(!confirmPasswordVisible);
+	};
+
+	return (
+		<div className="flex flex-col gap-5 w-11/12">
+			<p className="text-4xl m-0 font-bold text-center max-[700px]:text-3xl">
+				Basic Information
+			</p>
+
+			<div className="flex gap-5 w-full max-[1000px]:flex-col max-[1000px]:items-center">
+				<div className="flex flex-col w-6/12 max-[1000px]:w-full">
+					<label className={`${styles.label}`} htmlFor="first-name">
+						First Name <RequiredAsterisk />
+					</label>
+					<input
+						className={`${styles.input}`}
+						type="text"
+						name="first-name"
+						id="first-name"
+						required
+					/>
+				</div>
+				<div className="flex flex-col w-6/12 max-[1000px]:w-full">
+					<label className={`${styles.label}`} htmlFor="last-name">
+						Last Name <RequiredAsterisk />
+					</label>
+					<input
+						className={`${styles.input}`}
+						type="text"
+						name="last-name"
+						id="last-name"
+						required
+					/>
+				</div>
+			</div>
+
+			<div className="flex gap-5 w-full max-[1000px]:flex-col max-[1000px]:items-center">
```

Comment:
repeating: another textfield component or with a prop to make it variable width size, which you can achieve since Tailwind is like inline styles